README.md

```diff
@@ -20,11 +20,25 @@ $ git clone --recursive https://github.com/pytorch/test-infra
 │       ├── download.pytorch.org
 │       └── metrics.pytorch.org
 ├── stats                                # CI related stats commited automatically by a bot
-└── tools                                # Tools and scripts
-    ├── clang-tidy-checks
-    └── scripts
+├── tools                                # Tools and scripts
+|   ├── clang-tidy-checks
+|   └── scripts
+└── torchci                              # Code for hud.pytorch.org and our pytorch bots which run there   
+    └── pages 
 ```
 
+## Setting up your Dev environment to locally run hud.pytorch.org
+1. Install yarn:
+    `brew install yarn`
+2. `cd torchci` and install dependencies with `yarn install`
+2. Setup your environment variables
+
+    a. Copy `torchci/.env.example` to `torchci/.env.local` to create a local copy of your envionrmnet variables.  This will NOT be checked into git
```

Comment:
```suggestion
    a. Copy `torchci/.env.example` to `torchci/.env.local` to create a local copy of your environment variables.  This will NOT be checked into git
```